### PR TITLE
fixed login form styles

### PR DESCRIPTION
**Pull request by aborenko1996**

### CODE REVIEW

File: frontend/src/components/auth/LoginForm.vue [hunk 1]
```diff
@@ -105,27 +105,12 @@
...
-                margin-top: 4px;
-            }
-        }
-
         &.v-input--is-focused {
-            border: 1px solid rgba(60, 87, 222, 0.52);
-            box-shadow: 0px 0px 14px rgba(194, 205, 223, 0.6);
+            ::v-deep {
+                .v-input__slot {
+                    border: 1px solid rgba(60, 87, 222, 0.52);
```
Comment (TatsuUkraine) [inline frontend/src/components/auth/LoginForm.vue]: you sure that you need this? vutify should already have border specified for inputs

Comment (aborenko1996) [reply frontend/src/components/auth/LoginForm.vue]: Without this property input will not have blue border on focus

Comment (TatsuUkraine) [reply frontend/src/components/auth/LoginForm.vue]: it seems that this going to be global styles, if so - it's better to define them in root component (where font definitions are placed)

**Comment by TatsuUkraine** [comment]
@aborenko1996 lest remove border color from this component. We will have separate ticket to style inputs globally

**Pull request merged**